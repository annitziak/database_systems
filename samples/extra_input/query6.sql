SELECT Student.A FROM Student WHERE Student.A >=2 AND Student.B <= 200;
