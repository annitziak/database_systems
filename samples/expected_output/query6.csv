1, 200, 50, 33, 101, 2, 3
4, 100, 50, 11, 101, 2, 3
1, 200, 50, 33, 102, 3, 4
4, 100, 50, 11, 102, 3, 4
1, 200, 50, 33, 104, 104, 2
4, 100, 50, 11, 104, 104, 2
1, 200, 50, 33, 103, 1, 1
4, 100, 50, 11, 103, 1, 1
1, 200, 50, 33, 107, 2, 8
3, 100, 105, 44, 107, 2, 8
4, 100, 50, 11, 107, 2, 8
